modules/custom/az_news/config/install/core.entity_view_mode.node.feature.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
This file should probably exist in https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_core/config/install with the other display modes